widget/radio.go

```diff
@@ -20,6 +20,20 @@ type radioRenderer struct {
 	radio   *Radio
 }
 
+func removeDuplicatedOptions(options []string) []string {
+	var result []string
+	found := make(map[string]bool)
+
+	for _, option := range options {
+		if _, check := found[option]; !check {
```

Comment:
This is more idiomatically `if _, ok := found[option]; !ok {` (unless "check" has some domain specific meaning)
Also suggesting the function name `func dedupe([]string) []string`